Podfile

```diff
@@ -0,0 +1,15 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'DevCycle' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for DevCycle
+  pod 'LDSwiftEventSource', '~> 3.0'
```

Comment:
if you're using carthage in the example app, do we still need the `Podfile` and `Podfile.lock` here?

Reply:
I was using cocoapods in the example app

Reply:
the example app has it's own Podfile so we don't need this one

Reply:
oh this is the root one, thats weird I thought I removed this..